Yoast/wordpress-seo
# Downgrade React to 16.4.1 to match WordPress/Gutenberg

## Summary

This PR can be summarized in the following changelog entry:

* N/A

## Relevant technical choices:

* Downgrade React to version `16.4.1` to match Gutenberg's version. This is a temporary fix, because we shouldn't actually depend on our own version of React at all. See #11568

## Test instructions

This PR can be tested by following these steps:

* It is important you run `yarn install` and `grunt build:js` before testing this PR.
* Make sure #11563 is fixed.

## Quality assurance


Fixes #11563
